Libplanet.RocksDBStore/RocksDBStore.cs

```diff
@@ -20,7 +21,7 @@ namespace Libplanet.RocksDBStore
     /// This stores data in the RocksDB.
     /// </summary>
     /// <seealso cref="IStore"/>
-    public class RocksDBStore : BaseStore
+    public class RocksDBStore : BaseStore, IBlockStatesStore
```

Comment:
How about implementing `IBlockStateStore` onto `BaseStore` and renaming `BaseStore` to `BaseBlockStateStore`? (if then, we can skip some code changes in this PR)

Reply:
It looks good to me. I will apply for it. Thanks 🙏